ironmon_tracker/Languages/English.lua

```diff
@@ -509,6 +509,8 @@ ScreenResources{
 		ButtonShareSeed = "Share Seed",
 		CheckboxShowPreEvolutions = "Show Pre Evolutions",
 		CheckboxCustomTrainerNames = "Custom Trainer Names",
+		CheckboxOpenBookMode = "Open Book Play Mode",
+		LabelExtraTimeWarning = "Adds 2 seconds to game load",
```

Comment:
(This is the physicist in me coming out here) In what way was this 2 seconds measured, is it an estimate from one system or more an average observed across multiple?

Considering the huge variance of systems out there I feel it could be tough to quantify for certain, may be better to word it as something more vague like "Loading on new game will take slightly longer" (or whatever wording fits)

Reply:
I picked "2 seconds" as just a way to use less words to convey the meaning that enabling this option will add some extra startup time. I had trouble finding words that'd fit, especially if this will be translated.

2 seconds is more than 1 second, which means the extra time is not nothing. In reality (for my machine), it takes 1 second or less. So the "2 seconds" is more of a, if it goes faster thats great, but do expect some delay.

I'm up for using a different descriptive warning message here.

Reply:
fair, though the thing to keep in mind is that different people have different thought processes and interpretations of things. I think it'd be better to not give specific amounts (unless they can be quantified as such) and stick to a more general warning that it may increase startup time slightly, since that's ultimately what we're wanting them to expect

Reply:
How's this?

![image](https://github.com/besteon/Ironmon-Tracker/assets/4258818/1ccc371e-08df-4592-97ef-322386eb580a)

Reply:
hm, the "game load" part feels a little off (since it's technically the tracker loading not the game)
Does something like "May add extra load time" fit / work?

Reply:
I like that more, committed that. https://github.com/besteon/Ironmon-Tracker/pull/404/commits/30230de2e7fdc1b7f1700667c3cb235c47e8aaa8